### ISSUE TITLE
This plugin breaks disabling import rules

### ISSUE THREAD

**Issue by bodinsamuel** [alexgorbatchev/eslint-import-resolver-typescript#35]
When I install this plugin along `eslint-plugin-import` I can not use `eslint-disable import/<whatever>` 

For example, I used
`/* eslint-disable import/no-extraneous-dependencies */`

Eslint reports me:
` error  'import/no-extraneous-dependencies' rule is disabled but never reported  eslint-comments/no-unused-disable`

N.B: I extends the plugin and also list some rules:
```
extends: [
    'plugin:import/errors',
    'plugin:import/warnings',
    'plugin:import/typescript',
],
rules: {
  'import/no-extraneous-dependencies': ['error', { devDependencies: true }],
}
```

--- 
If I remove the `eslint-disable` then `import` correctly report an extraneous import, that means the rules is reported but someway eslint does recognise it 🤷🏻‍♂️

**Comment by JounQin** [comment]
It seems you didn't enable this resolver even. 
And a runnable minimal reproduction is *required*.
